spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/GravitinoSparkConfig.java

```diff
@@ -13,5 +13,19 @@ public class GravitinoSparkConfig {
   public static final String GRAVITINO_HIVE_METASTORE_URI = "metastore.uris";
   public static final String SPARK_HIVE_METASTORE_URI = "hive.metastore.uris";
 
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_BACKEND = "catalog-backend";
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_TYPE = "type";
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_WAREHOUSE = "warehouse";
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_URI = "uri";
+  public static final String GRAVITINO_JDBC_USER = "jdbc-user";
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_JDBC_USER = "jdbc.user";
+  public static final String GRAVITINO_JDBC_PASSWORD = "jdbc-password";
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_JDBC_PASSWORD = "jdbc.password";
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_JDBC_INITIALIZE = "jdbc-initialize";
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_JDBC_DRIVER = "jdbc-driver";
+
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_BACKEND_HIVE = "hive";
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_BACKEND_JDBC = "jdbc";
```

Comment:
I was thinking that should we 1) define a IcebergPropertiesConstans like #2605 ; b) use the configuration defined in bundled-catalog, not define again here. WDYT? @FANNG1

Reply:
yes, #2605 introduced the `bundle-catalog` dependences,  we could define `IcebergPropertiesConstants` to reference properties from `bundle-catalog` like `IcebergPropertiesConstants`

Reply:
updated. Thank you.